frontend/src/i18n/index.ts

```diff
@@ -1,36 +1,50 @@
 import { I18nOptions } from 'vue-i18n';
-import de from './de-DE.json';
-import en from './en-US.json';
-import fr from './fr-FR.json';
-import it from './it-IT.json';
-import ko from './ko-KR.json';
-import nl from './nl-NL.json';
-import pt from './pt-PT.json';
-import tr from './tr-TR.json';
+import deDe from './de-DE.json';
+import enUs from './en-US.json';
+import frFr from './fr-FR.json';
+import itIt from './it-IT.json';
+import koKr from './ko-KR.json';
+import lvLv from './lv-LV.json';
+import nlNl from './nl-NL.json';
+import ptBr from './pt-BR.json';
+import ptPt from './pt-PT.json';
+import ruRu from './ru-RU.json';
+import trTr from './tr-TR.json';
+import uaUa from './uk-UA.json';
+import zhTw from './zh-TW.json';
 
 import { createI18n } from 'vue-i18n';
 
-// ISO 639‑1 two letter code
 export enum Locale {
-  EN = 'en',
-  DE = 'de',
-  FR = 'fr',
-  IT = 'it',
-  KO = 'ko',
-  NL = 'nl',
-  PT = 'pt',
-  TR = 'tr'
+  ENUS = 'en-US',
+  DEDE = 'de-DE',
+  FRFR = 'fr-FR',
+  ITIT = 'it-IT',
+  KOKR = 'ko-KR',
+  LVLV = 'lv-LV',
+  NLNL = 'nl-NL',
+  PTBR = 'pt-BR',
+  PTPT = 'pt-PT',
+  RURU = 'ru-RU',
+  TRTR = 'tr-TR',
+  UKUA = 'uk-UA',
+  ZHTW = 'zh-TW',
```

Comment:
I would prefer `EN_US`, or even violate constant naming rules and use `en_US`.

Reply:
Tried it before but had linter problems, now not anymore :thinking: , fixed in [c14894b](https://github.com/cryptomator/hub/pull/350/commits/c14894bb06da880d067fad203feec2d8838aaf3c)